ros1-bridge-ws/docker/debug/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  ros1-bridge-debug:
+    build: .
+    image: yuzhong1214/ros1-bridge:debug
+    container_name: ros1-bridge-debug
+    stdin_open: true
+    tty: true
+    privileged: true
+    command: ./start-bridge.sh
+    network_mode: host
+    working_dir: /home/user
```

Comment:
The `working_dir` is different from `docker/docker-compose.yaml` (which is `/`), I wonder if this is intended.